manual/src/cmds/debugger.etex

```diff
@@ -4,14 +4,6 @@
 This chapter describes the OCaml source-level replay debugger
 "ocamldebug".
 
-\begin{unix} The debugger is available on Unix systems that provide
-BSD sockets.
-\end{unix}
-
-\begin{windows} The debugger is available under the Cygwin port of
-OCaml, but not under the native Win32 ports.
-\end{windows}
```

Comment:
Maybe mention here that the "replay" facility is not supported on Windows.

Reply:
What is the reason for "replay" not being available on Windows?

Reply:
As [mentioned](https://github.com/ocaml/ocaml/pull/13087#pullrequestreview-1990920068), this is due to its dependence on `fork`.